README.md

```diff
@@ -24,19 +24,30 @@ module "admin_tier" {
   stage                       = "${var.stage}"
   subnets                     = ["${var.subnets}"]
   zone_id                     = "${module.tf_domain.zone_id}"
+  security_groups             = ["${var.security_groups}"]
+  associate_public_ip_address = "${var.associate_public_ip_address}"
 }
 
 ```
 
 This will create a `id`, `fqdn`, `security_group_id`, `role` and `public_ip`.
```

Comment:
`id`, `fqdn`, `security_group_id`, `role` and `public_ip` are outputs, but not created resources.
Delete it, as we have `tf_admin` purpose description at the top and outputs description at the bottom.